linux_test.go

```diff
@@ -4,9 +4,9 @@ package lumberjack
 
 import (
 	"os"
+	"sync"
```

Comment:
What's the easiest way to test the linux version?

Reply:
add a build tag for only running on linux (if necessary), then run on a linux machine (e.g. Jenkins)